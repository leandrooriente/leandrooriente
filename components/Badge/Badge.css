.badge {
  color: var(--secondary-color);
  display: inline-block;
  font-size: 14px;
  text-decoration: none;
}
