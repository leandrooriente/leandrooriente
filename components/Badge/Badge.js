import React from 'react';
import PropTypes from 'prop-types';
import css from './Badge.css';

const Badge = ({ children, className }) => {
  const classes = `${css.badge}${className ? ` ${className}` : ''}`;

  return (
    <span className={classes}>
      { children }
    </span>
  );
};

Badge.defaultProps = {
  className: null,
};

Badge.propTypes = {
  className: PropTypes.string,
  children: PropTypes.node.isRequired,
};

export default Badge;
