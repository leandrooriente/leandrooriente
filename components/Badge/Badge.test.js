import React from 'react';
import { mount } from 'enzyme';
import Badge from './Badge';

describe('<Badge />', () => {
  it('should render <Badge /> component', () => {
    const wrapper = mount(
      <Badge>
        Test
      </Badge>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
