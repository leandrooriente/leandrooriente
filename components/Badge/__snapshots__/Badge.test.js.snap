// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> should render <Badge /> component 1`] = `
<Badge
  className={null}
>
  <span
    className="badge"
  >
    Test
  </span>
</Badge>
`;
